dashboard/app/models/user_level.rb

```diff
@@ -18,14 +19,14 @@
 #
 
 # Summary information about a User's Activity on a Level in a Script.
-# Includes number of attempts (attempts), best score and whether it was submitted (best_result)
+# Includes number of attempts (attempts), best score and whether it was submitted
 class UserLevel < ActiveRecord::Base
   belongs_to :user
   belongs_to :level
   belongs_to :script
 
   def submitted?
```

Comment:
I think ActiveRecord defines this method now so defining it ourselves should be unnecessary.